src/main/resources/application.properties

```
@@ -85,6 +85,9 @@ puppetpdf.port=${GOVER_PUPPET_PDF_PORT}
 # Set the encryption key for the secrets
 secrets.key=${GOVER_SECRETS_KEY}
 
+# Set the encryption key for captcha challenges
+captcha.key=${GOVER_CAPTCHA_KEY:}
```

wenn der key nicht optional ist, dann sollte hier ein fehler fliegen
```suggestion
captcha.key=${GOVER_CAPTCHA_KEY}
```